packages/test/test-end-to-end-tests/src/test/rootDatastores.spec.ts

```diff
@@ -119,7 +115,6 @@ describeNoCompat("Named root data stores", (getTestObjectProvider) => {
 
     describe("Aliasing", () => {
```

Comment:
The last `it` test in this file `"Assign multiple data stores to the same alias, first write wins, "` probably should be moved out of the describe block as it calls `setupContainers` twice which calls `makeTestProvider` twice.